scripts/external_ips_scripts/get-network-policy.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+ROX_ENDPOINT=${1:-localhost:8000}
+
+clusters_json="$(curl --location --silent --request GET "https://${ROX_ENDPOINT}/v1/clusters" -k --header "Authorization: Bearer $ROX_API_TOKEN")"
+
+cluster_id="$(echo "$clusters_json" | jq -r '.clusters[0].id')"
+
+query="Cluster%3Aremote%2BNamespace%3Aqa&includePorts=true"
```

Comment:
To avoid doing the url encoding ourselves we can use curl 

```
curl --data query="Cluster:remote+Namespace:qa" \
     --data includePorts=true \
     --data deleteExisting=NONE \
     ...
```